src/Application/Actions/Donations/Update.php

```diff
@@ -101,21 +83,27 @@ protected function action(Request $request, Response $response, array $args): Re
         }
 
         if (!isset($donationData->status)) {
-            $this->entityManager->rollback();
```

Comment:
same here and above - what changes needed rolling back? So deleting these lines looks good.